### ISSUE TITLE
Output text file with lat/lon corner points needed for RRFS fire weather run

### ISSUE THREAD

**Issue by EricRogers-NOAA** [NOAA-EMC/UPP#530]
In INITNEMS.f, there is this logic for the NMMB:

          IF (MAPTYPE==205)THEN  !A STAGGERED B-GRID
            open(112,file='latlons_corners.txt',form='formatted' &
             ,status='unknown')
            if(grib=="grib2") then
              write(112,1001)LATSTART/1000,(LONSTART/1000)-360000, &
                  LATSE/1000, &
                  LONSE/1000,LATNW/1000,LONNW/1000,LATLAST/1000, &
                  (LONLAST/1000)-360000
            endif
1001        format(4(I6,x,I7,x))
          close(112)
          ENDIF

The above logic writes out the earth lat/lon of the 4 corner points of the NAM b-grid into the "latlons_corners.txt" file. For the NAM fire weather nest (which can be placed anywhere in the CONUS or Alaska) post-processing, this file is used by the code that creates the WGRIB2 grid specs for the fire weather output grid (lambert conformal over CONUS or polar stereographic over Alaska).

I am working on a 1.5 km RRFS fire weather domain which will replace the NAM fire weather nest. I need the same code fragment placed in current UPP for the regional FV3, presumably in INITPOST_NETCDF.f. The RRFS fire weather run will output (via the inline post) a rotated lat/lon grid, and by having the lat/lons of the 4 corner points of the rotated lat/lon grid, I can calculate the grid specs for the output grid as is now done for the NAM's fire weather nest.

**Comment by WenMeng-NOAA** [comment]
@EricRogers-NOAA 
My test results for off-line post is at /u/wen.meng/ptmp/fv3r_2022030200 on cactus
And the results for in-line post at /u/wen.meng/ptmp/FV3_RT/rt_60471/regional_control on cactus.
Do you also need output of lat/lon on four points for other than rotated lat-lon grids?

**Comment by EricRogers-NOAA** [comment]
Hi Wen,

Sorry, I made an error in how I worded this sentence "The RRFS fire weather run will output (via the inline post) a rotated lat/lon grid, and by having the lat/lons of the 4 corner points of the rotated lat/lon grid, I can calculate the grid specs for the output grid as is now done for the NAM's fire weather nest."

What I meant to say was this "The RRFS fire weather run will output (via the inline post) a rotated lat/lon grid, and by having the 
**earth coordinate lat/lon** of the 4 corner points of the rotated lat/lon grid, I can calculate the grid specs for the output grid as is now done for the NAM's fire weather nest."

In other words I need the lat/lon of the 4 corner points converted to earth coordinates in that text file. Sorry for the misunderstanding.

**Comment by WenMeng-NOAA** [comment]
@EricRogers-NOAA I modified the changes. The latlons_corners.txt has:
```
-33000 309000  33000  51000
```
Would these be ones you expect? My new test is at /u/wen.meng/ptmp/fv3r_2022030200 on cactus.

**Comment by EricRogers-NOAA** [comment]
I'm afraid not. First, these are the rotated lat/lon values, not earth lat/lon values. The attached map shows the PRSLEV and NATLEV domain ("RRFS Rotated Lat/Lon") projected onto its actual location on the Earth. The NE corner point in Spain is at 41.48N, 358.2E, that is number I need. Also, what is listed is just the SW and NE corner point values, what is done for the NAM post and what is needed is all 4 lat/lon corner values. Thanks
![rrfs_domain](https://user-images.githubusercontent.com/48292889/178340140-a1a2914f-faeb-480c-8a63-2fd00b0da689.jpg)
s

**Comment by WenMeng-NOAA** [comment]
@EricRogers-NOAA Thanks for explaining. I modified the changes and tested with a case of 3-km North America domain. Please see the results at  /u/wen.meng/ptmp/post_fv3r_2022033112.
```
wen.meng@clogin03 post_fv3r_2022033112$ cat latlons_corners.txt
 -1605 202671  -1605 291328  41479 135802  41479 358197
```

**Comment by EricRogers-NOAA** [comment]
Yes, that is what I was looking for. Thanks! Could you point me at the code change for in the inline post? I've been making RRFS fire weather runs and would like to test it.

**Comment by WenMeng-NOAA** [comment]
@EricRogers-NOAA Thanks for confirming. I might consult with Jun for the changes in in-line post since lat/lon start/last are not available in the interface.

**Comment by EricRogers-NOAA** [comment]
OK, I'm good with using the offline post with this change for now.

**Comment by WenMeng-NOAA** [comment]
@EricRogers-NOAA You may use my working branch for off-line post testing at
https://github.com/WenMeng-NOAA/UPP/tree/post_rrfs_latlon

**Comment by EricRogers-NOAA** [comment]
Thank you; it worked fine for me, making the text file.
